ChangeLog.md

```diff
@@ -4,29 +4,244 @@ Notable changes to the FreeOrion project will be documented in this file.
+   
+- Production
+    - Fixed erroneous early projected completion times for production queue items with multiple repetitions.
+    - Added commands to split or duplicate production items on the queue.
+    - If the selected system is a production item's rally location, this is indicated on the item in the queue.
+
+- Design screen
+    - Ship designs can be reordered, which will affect their ordering on the production screen.
+    - Removed dialog for saving ship designs; a right click popup command is used instead.
+    - Added right-click popup commands to obsolete parts, hulls, or ship designs, or to delete designs.
+    - Added buttons to toggle part / hull obsolete filters.
+    - Indicated part / design availability and obsolescence with desaturated borders in lists.
+    - Added prompt to toggle availability filters when the hulls list is empty.
+    - Added prompt to saved designs to suggest adding some.
+    - Fixed bug where replacing a design twice started added designs.
+    - Allowed monster designs to be viewed, edited, and saved.
+    - When dragging a part over a slot, a part already in the slot should be hidden unless the part in the slot is being dragged.
+    - Ctrl-Dragged parts will remove or replace all parts of the same type.
+    - Removed restriction on setting blank design descriptions.
+
+- Multiplayer
+    - Multiplayer lobby remembers previously-used settings when returning to the lobby after a game.
+    - Added option to limit manipulation of settings in multiplayer lobby to the host.
+    - Added server player name / passwords list, which can require players to password authenticate to join a server.
+        - Players may be assigned roles, which control what they can do in the lobby.
+    - Added optional timestamps to chat in multiplayer lobby and in-game chat.
+    - Ready button in multiplayer lobby will lock if not enough players, and unready if settings were changed by another player.
+    - Added choice of client type in the network connection window.
+    - Disambiguated "Player" connection type from "Human", which is also an in-game species.
+    - Allowed joining an game being played as a moderator or observer.
+    - Split Resign button into a Concede button and a Resign button.
+       - Conceding from a multiplayer game, when conditions allow it, eliminates the empires' assets from the universe.
+    - Hid load button in multiplayer games.
+
+- Misc
+    - Improved objects list filter dialog layout.
+    - Added granular log detail-level options for specific game systems and clients / sever
+    - Made some droplists not respond to mousewheel events when not open (dropped), so that they won't be unintentionally manipulated.
+    - Added options for whether to auto-add default or saved designs to the player's empire.
+    - Game will auto-save when quitting or resigning. This can be aborted during the save by the user.
+    - Added options to enable autosaves at start and end of turns.
```

Comment:
I think this needs to be clarified someplace at least, because having looked into it more with Morlic, this basic description seems misleading.  Or perhaps we leave the description as-is and consider the current implementation to be buggy and intend to fix it.  The issue is that both the turn-start and turn-end autosaves will be done after, and include, the AI orders for that turn.  The only difference is that the turn-start autosave does not include the user-player orders, and the turn-end autosave does.

Reply:
Technically that's correct of course, but from a player's perspective it still looks like the one savegame happens at the start of the turn, and the other at the end. As a player I don't really care if the AI orders are already in the "start turn" savegame or not, I only care about *my* turn (orders).

Looking at it from that angle the statement isn't entirely incorrect. So I propose to leave it as it is, and go with your second suggestion: consider the current state as something we intend to fix in the future, and the wording of this statement as how things are intended to be.